packages/gatsby-theme/tsconfig.json

```diff
@@ -21,5 +21,5 @@
     "baseUrl": "."
   },
   "exclude": ["node_modules"],
-  "include": ["./src/**/*.tsx", "./src/**/*.ts", "./images.d.ts"]
+  "include": ["./src/**/*.tsx", "./src/**/*.ts", "./images.d.ts", "src/utils/WithCurrentUser.js"]
```

Comment:
How come we made `WithCurrentUser ` into a .js file?

Reply:
@rohit-ravikoti I need help making it a .ts file